indexer/locals.tf

```diff
@@ -148,15 +163,10 @@ locals {
             name : "BLOCKED_ADDRESSES",
             value : var.indexer_compliance_blocklist,
           },
-          {
-            name : "PG_POOL_MAX",
-            value : "2"
-          },
-          var.roundtable_ecs_environment_variables,
         ],
       ),
     },
-    "vulcan" : {
+    "${local.service_names["vulcan"]}" : {
```

Comment:
What's the purpose of this?

Reply:
To make it clear that the service names are from the locals variable `service_names`.